### PR TITLE
implement parsing custom user-defined property labels

### PR DESCRIPTION
**Pull request by nugthan**
Currently, the library does not extract any of the user-defined property group labels (if you right-click on the group in After Effects, select "rename"). This merge adds parsing of the custom properly labels, a custom label to one of the test files, and a test case.

### CODE REVIEW

File: property.go [hunk 1]
```diff
@@ -106,6 +107,19 @@ func parseProperty(propData interface{}, matchName string) (*Property, error) {
...
 			}
+			tdgpBlock, err := propHead.SublistFind("tdgp")
+			if err == nil {
+
+				// Look for a tdsn which specifies the user-defined label of the property
+				tdsnBlock, err := tdgpBlock.FindByType("tdsn")
+				if err == nil {
+					label := fmt.Sprintf("%s", tdsnBlock.ToString())
+
+					if label != "-_0_/-" {
```
Comment (rioam2) [inline property.go:120]: Does this label have any specific special meaning? If so, a comment explaining the significance of `-_0_/-` would be good to document here

**Review by rioam2 — APPROVED** [review]
Thanks for your contribution! Would it make sense to have the `Label` field default to be the same as `Name`? 

Changes look great to me. I've made a few remarks in my review. If you have a few minutes to address them, I can merge afterwards!

**Comment by nugthan** [comment]
Have added some documentation - For some reason, in adobe's great wisdom it defaults to "-_0_/-" if the user hasn't defined a custom name for an effect.

**Comment by rioam2** [comment]
> Have added some documentation - For some reason, in adobe's great wisdom it defaults to "-_0_/-" if the user hasn't defined a custom name for an effect.

I'm anticipating this is an encoding of some other unknown data (which happens to be `-_0_/` in ASCII). For now, it doesn't look very important, so let's just ignore it ;)

Thanks again for your contribution, and happy hacking!

**Pull request merged**